### PR TITLE
add test_rclcpp package testing subscriptions and spinning for now

### PR DESCRIPTION
**Pull request by dirk-thomas**
The test currently fails with all rmw implementations.

### CODE REVIEW

**Comment by esteve** [comment]
+1

File: test_rclcpp/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  if(NOT WIN32)
+    set(test_executable_extension "")
+    set(test_executable_subfolder "")
+  else()
+    set(test_executable_extension ".exe")
+    set(test_executable_subfolder "/${CMAKE_BUILD_TYPE}")
+    if("${test_executable_subfolder}" STREQUAL "/")
+      set(test_executable_subfolder "/Debug")
+    endif()
+  endif()
```
Comment (wjwwood) [inline test_rclcpp/CMakeLists.txt]: Where are these variables used?

Comment (dirk-thomas) [inline test_rclcpp/CMakeLists.txt]: Nowhere. I removed them. Thanks.

**Comment by jacquelinekay** [comment]
Perhaps the timeout tests from ros2/system_tests/pull/19 should be in this package rather than `test_communication`. I can revisit that PR to put the test in `test_rclcpp` after this one gets merged if you agree.

**Comment by dirk-thomas** [comment]
@jacquelinekay I agree the `test_rclcpp` is the better location for testing parameters.

**Pull request merged**